app/services/data_set_documentation.rb

```diff
@@ -2,35 +2,27 @@ class DataSetDocumentation
   def self.for(klass)
     name = klass.name.demodulize.underscore
 
-    begin
-      spec = YAML.load_file(Rails.root.join("app/exports/#{name}.yml"))
-    rescue Errno::ENOENT
-      return false
```

Comment:
This will now raise the `Errno::ENOENT` error instead if there is a doc file missing